src/uct/cuda/cuda_copy/cuda_copy_md.h

```diff
@@ -32,6 +32,7 @@ typedef struct uct_cuda_copy_md {
         int                     dmabuf_supported;
         uct_cuda_pref_loc_t     pref_loc;
     } config;
+    int cuda_ctx_retained;
```

Comment:
move it to after 'super' and align on column

Reply:
Fixed.